Lib/test/test_asyncgen.py

```diff
@@ -393,6 +393,131 @@ async def gen():
                 r'anext\(\): asynchronous generator is already running'):
             an.__next__()
 
+    def test_async_gen_asend_throw_concurrent_with_send(self):
```

Comment:
[Ignore this review, I'm testing if Bedevere will run without needing approval.]